pkg/workflows/wasm/host/test/runnerapi/cmd/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/cli/cmd/testdata/fixtures/capabilities/basictrigger"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
+)
+
+func BuildWorkflow(config []byte) *sdk.WorkflowSpecFactory {
```

Comment:
Thanks @cedric-cordenier for the change. The UX is a lot neater:
1. BuildWorkflow() can stay in the main.go, and doesnt need to be in a separate file.
2. The main.go doesnt need to have the wasip1 build constraint, actually no constraint needed at all.